modules/ROOT/pages/salesforce/salesforce-connector-studio.adoc

```diff
@@ -0,0 +1,165 @@
+= Salesforce - Studio
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+== Install Salesforce  
```

Comment:
Install Salesforce Connector? Why? (because it has to be in Studio before you can add it to an app flow, yes?)